pages/hpc/jobs.md

```diff
@@ -170,11 +170,24 @@ supervision or interaction. Batch jobs are commonly used for applications that
 run for long periods of time and require no manual user input.
 
 Below is an explanation of the SBATCH options used in our samples. These are
-Slurm directives and should be understood before submitting a job.  For more
-information on Slurm directives, partitions, and QOS, [see here](#slurm-directives-partitions-qos).
+called "Slurm directives" and should be understood before submitting a job.  For more
+information on clusters, partitions, and QOS options, see [here](#slurm-directives-partitions-qos).
 
 ```
 #!/bin/bash -l
+
+# 	Tell Slurm which cluster, partition and qos to use to schedule this job
+
+#SBATCH --cluster=cluster-name
+#SBATCH --partition=partition-name
+#SBATCH --qos=qos-name
+
+# 	Tell Slurm which account to run this job under 
+#   If not specified, your default account will be used  
```

Comment:
The format of white space between line 185-186 isn't the same in the text document, but they show identically on Mkdocs. There's a few other comments in this code block that differ in the same way.

Reply:
Yeah this is due to 2 different people writing these.  I've fixed to align with original author's formatting